monai/bundle/scripts.py

```diff
@@ -1191,10 +1191,10 @@ def onnx_export(
 
 
 def ckpt_export(
-    net_id: str | None = None,
-    filepath: PathLike | None = None,
-    ckpt_file: str | None = None,
-    meta_file: str | Sequence[str] | None = None,
+    net_id: str | None = "network_def",
+    filepath: PathLike | None = "models/model.ts",
```

Comment:
```suggestion
    filepath: PathLike | None = None,
```
is it possible to still use `None` here? and then the default logic could be:
```
meta_file_ = os.path.join(bundle_root_, "configs", "metadata.json") if meta_file_ is None " else meta_file_
```

if None doesn't work we can consider `""`

Reply:
Sure, updated it in the latest commit.